Document vitals sensor subsystem design details

jflores43 opened the pull request:
Added detailed design information for the vitals sensor subsystem, including its function, constraints, specifications, proposed solution, interface with other subsystems, PCB layout, BOM, and analysis.

jflores43 committed:
Document vitals sensor subsystem design details

Added detailed design information for the vitals sensor subsystem, including its function, constraints, specifications, proposed solution, interface with other subsystems, PCB layout, BOM, and analysis.

jflores43 committed:
Add temp.md file with initial content

jflores43 committed:
Delete Reports/Detailed Design/Vital Sense directory

jflores43 committed:
Add temp file to Vital Sense Images directory

jflores43 committed:
added images for Detail Design

removed reliance on previous team's repo for Vital Sense detail design images

jflores43 committed:
Delete Reports/Detailed Design/Vital Sense Images/temp

jflores43 committed:
Revise Sensors Subsystem documentation and images

Updated descriptions and images for the Sensors Subsystem. Clarified the function of the subsystem and provided new links to images.

jflores43 committed:
Revise Sensors Subsystem documentation and BOM

Clarified subsystem function and updated BOM with alternative sensor options and costs.

jflores43 committed:
Update EIRP limit to 40 dBm for FCC compliance

I think this is the last change that I found that I needed. . . surely. . .

csjohnson43 reviewed (changes requested):
- For each figure, a brief description is needed. Expand upon the conceptual schematic and operational flowchart a bit
- List the size and weight of the radar chip to prove it meets constraints.
- Explain the parameters in Equation (2)
- Need to show pseudocode or prototype code in the document. I would start prototyping some since we have the chip already. 

After addressing these comments, you have my approval!

jflores43 committed:
Revise Sensors Subsystem documentation and references

Updated sensor subsystem documentation, addressed advised adjustments: 
- brief descriptions added for each figure
- expansion on conceptual schematic and operational flow chart
- system dimensions listed to show it fits constraints
- high-level pseudocode added

jflores43 commented on Reports/Detailed Design/Sensors Subsystem.md:
```diff
@@ -83,18 +104,57 @@ The solution is low-cost and small enough, both in volume and in mass, to fit th
 </p>
-  
+
+The equation above outlines how EIRP of the sensor is the sum of the power consumed by the transmitter and the gain of the transmitter's antenna. Losses are also taken into account by subtracting them from the sum. All quantities are in decibels.
```
forgot to say I added this part too
